packages/amplication-server/.env

```diff
@@ -62,7 +62,7 @@ BITBUCKET_CLIENT_SECRET=[bitbucket-client-secret]
 BILLING_ENABLED=false
 BILLING_API_KEY="123456"
 
-CONTACT_US_LINK="https://meetings-eu1.hubspot.com/liza-dymava/cta-link"
```

Comment:
was this updated in the cloud enviroments?

Reply:
not yet as much as I know but I told him that we need to do it

Reply:
https://github.com/amplication/amplication-cluster-configuration/pull/168